.github/workflows/CI.yml

```diff
@@ -23,18 +23,26 @@ jobs:
 
     - uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: | 
+        dotnet-version: |
           3.1.x
           6.0.x
 
     - run: dotnet --info
 
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v2
+      with:
+        languages: csharp, javascript
```

Comment:
Do we need javascript here? No JS files in PALT

Reply:
removed JS. Good catch!